### PR TITLE
Button.cs documentation and formatting updates

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
Cleans up Button.cs a bit, in preparation for a fix for #1879 incoming after merge.

Changes
---
- First PR towards a fix for #1879 (splitting up into smaller PRs for review ease).

### CODE REVIEW

File: Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs [hunk 1]
```diff
@@ -93,29 +97,55 @@ public bool RequireGaze
         /// <summary>
         /// Last state of hands being visible
         /// </summary>
-        private bool _bLastHandVisible = false;
+        private bool lastHandVisible = false;
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs]: Just keep in mind that renaming these will break the previous serialization people have in their projects.

Comment (StephenHodgson) [reply Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs]: [There is an attribute we should be using to make sure it doesn't break](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html).
```
[FormerlySerializedAs("myValue")]
private string myValue;
```

Comment (keveleigh) [reply Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs]: What serialization does this break? None of the names I changed have the `[SerializeField]` tag, or are you referring to something else?

Comment (keveleigh) [reply Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs]: Good to know about that `[FormerlySerializedAs]` tag! That seems very useful.

Comment (StephenHodgson) [reply Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs]: Oh, I thought some were public before, then it's no worries.

**Pull request merged**